src/interfaces.ts

```diff
@@ -24,14 +24,14 @@ import { fetch as crossFetch } from "cross-fetch";
 export type protocols = "ws" | string;
 export type statuses = "connecting" | "connected" | "closing" | "closed";
 
-/** @internal */
+/** @hidden */
```

Comment:
Using `@internal` will result in types not being generated, not just in documentation not being generated; Our E2E tests for node.js access some of these "internal" properties, so instead we should just make them undocumented for now, instead of private/internal